eve_sqlalchemy/tests/config/resourceconfig/schema.py

```diff
@@ -22,6 +22,8 @@ class SomeModel(Base):
     a_pickle = Column(types.PickleType)
     a_string = Column(types.String(42), default='H2G2')
     _internal = Column(types.Integer)
+    a_nullable_boolean = Column(types.Boolean, nullable=True)
+    a_server_default_col = Column(types.Integer, server_default='0')
 
```

Comment:
It looks strange to use an integer but pass a string as default; the SQLAlchemy docs use the `text` function here. Maybe just use `types.String` as the type doesn't really matter here.